react_practice/react-apps/waitlist-page-v1/src/PrimaryBtn.css

```diff
@@ -1,21 +1,17 @@
-.primary-btn-title {
+.primary-btn {
+  border: 2px solid #1e90ff;
   padding: 10px 18px;
   background-color: #1e90ff;
   color: #fff;
   font-size: 12px;
   font-weight: 700;
-  max-width: 48%;
   border-radius: 5px;
   text-align: center;
-  margin: 10px auto 50px;
 }
 
-.primary-btn-title:hover {
-  padding: 8px 16px;
-  border: 2px solid black;
+.primary-btn:hover {
+  border-color: black;
   background-color: #fff;
   color: black;
-  font-size: 12px;
-  font-weight: 800;
```

Comment:
I removed the font-weight rule here on hover. This causes the button to jump a bit as this ever so slightly makes the text a bit bigger, which means the button also grows in size. By leaving the font weight the same, you can avoid the jump in size between hovering and not hovering.